api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java

```diff
@@ -378,6 +378,12 @@ public CohortDefinition getAgeOnArtStartDate(
     return cd;
   }
 
+  /**
+   * Age should be calculated on Patient ART Start Date (Check Section A for the algorithm to define
+   * this date).
+   *
+   * @return CohortDefinition
+   */
```

Comment:
Please include the parameters that are passed in the method in the javadoc, look at this as an example https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java#L96-L106

Reply:
and if there are any tests written for any of the methods, they need to be documented there as well have a look at this method https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/PatientService.java#L72-L83

Reply:
Okay, I'll follow the examples.